packages/oas-utils/src/entities/spec/spec-objects.ts

```diff
@@ -42,7 +42,7 @@ export const oasContactSchema = z.object({
  */
 export const oasInfoSchema = z.object({
   /** REQUIRED. The title of the API. */
-  title: z.string().optional().default('OpenAPI Spec'),
+  title: z.string().catch('API'),
```

Comment:
It’s required, but has a fallback for any invalid/undefined value.

Reply:
Oh so optional + default actually has to do with the types. Optional means you don't need to pass it in the input, default means it always exists after parse. Maybe we just slap the catch on there? Not sure how it affects the types

Reply:
https://www.gybex.dev/blog/zod-default-and-catch-methods

Reply:
I don't think adds any value in this case according to the blog

Reply:
alright yea I confirmed it, with .catch alone we don't get the proper types on output. we should use both `.catch` and `.default`